test/e2e_new/main_test.go

```diff
@@ -36,6 +37,7 @@ import (
 	"knative.dev/reconciler-test/pkg/environment"
 
 	"knative.dev/eventing-kafka-broker/control-plane/pkg/kafka"
+	brokertest "knative.dev/eventing-kafka-broker/test/pkg/broker"
```

Comment:
This additional import might be causing the above error because of the init function's side effects.

Reply:
Thanks for catching that. This is the flaky test effect: I overlooked that there's actually a problem!